src/pub_verif_token.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // Licensed under the Apache-2.0 license found in the LICENSE file or at https://opensource.org/licenses/Apache-2.0
 
-import { type BlindRSA, SUITES } from '@cloudflare/blindrsa-ts';
+import { type BlindRSA, RSABSSA } from '@cloudflare/blindrsa-ts';
+import type { BlindRSAPlatformParams } from '@cloudflare/blindrsa-ts/lib/src/blindrsa.js';
```

Comment:
since the constructor for a Suite was modified and now requires to pass (non-optional) arguments, then it must be exposed.

```diff
- const suite = BLIND_RSA.suite[this.mode]();
+ const suite = BLIND_RSA.suite[this.mode](this.params);
```

Reply:
there is a default on suite function, so no need to pass it https://github.com/cloudflare/blindrsa-ts/blob/main/src/index.ts#L63

Reply:
@armfazh the type can be exposed by merging and cutting a release for https://github.com/cloudflare/blindrsa-ts/pull/13

Reply:
updated.